session.go

```diff
@@ -4,43 +4,40 @@ import (
 	"reflect"
 	"sync"
 
-	"upper.io/db"
+	"upper.io/db.v2"
+	"upper.io/db.v2/postgresql" // TODO: Figure out how to remove this and make it generic.
```

Comment:
Main problem here is that the connection is now initialized on the `postgresql` package, we also have SQL-specific methods there. Perhaps we could add an bond-specific interface for this but that would require passing the adapter with some awkward function.